web/src/helpers.tsx

```diff
@@ -51,6 +51,11 @@ export const getMenuData = (): MenuDataItem[] => {
       path: '/consumer/list',
       icon: <IconFont name="iconconsumer" />,
     },
+    {
+      name: 'service',
+      path: '/service/list',
+      icon: <IconFont name="iconconsumer" />,
```

Comment:
do we need to add a new icon for service?

Reply:
oh yes, please select an icon here[1], then I will help to generate a new icon file.

[1] https://www.iconfont.cn/

Reply:
how about https://www.iconfont.cn/search/index?q=service-list

Reply:
fine, if there doesn't have a more proper icon, we could use it. I will file a new PR to do this.